mesa_geo/visualization/modules/MapVisualization.py

```diff
@@ -68,15 +69,20 @@ def __init__(
             of the space. Default is None.
         :param map_width: The width of the map in pixels. Default is 500.
         :param map_height: The height of the map in pixels. Default is 500.
+        :param scale_options: A dictionary of options for the map scale. Default is None (no map scale).
+            The available options can be found at: https://leafletjs.com/reference.html#control-scale-option
         """
         self.portrayal_method = portrayal_method
         self._crs = "epsg:4326"
 
-        if view is None and zoom is None:
-            new_element = f"new MapModule(null, null, {map_width}, {map_height})"
-        else:
-            new_element = f"new MapModule({view}, {zoom}, {map_width}, {map_height})"
-        self.js_code = f"elements.push({new_element});"
+        view_js = "null" if view is None else view
+        zoom_js = "null" if zoom is None else zoom
+        scale_options_js = "null" if scale_options is None else scale_options
```

Comment:
It'd be clearer if the conversion to JS object (`True` -> `true`, `False` -> `false`) happens here.

Reply:
Changed!